db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStorageServices < ActiveRecord::Migration[5.2]
+  def change
+    create_table :storage_services do |t|
+      t.string :name
+      t.string :description
+      t.string :uuid
+      t.bigint :version
```

Comment:
Is this always going to be an integer version or should we hedge and use a string field (I'm thinking weird versions like `1.0a-beta2`)

Reply:
Yes. This will be a incremental integer value to keep track of changes to a service. The idea is that capabilities in a storage service may be changed in the course of time and once changed a new service with a higher value will be created. This way existing resources will be associated with an older version of the service, and any new resources will be associated with the new service with a higher version.